pnet_sys/rls.toml

```diff
@@ -0,0 +1,7 @@
+# rls.toml
```

Comment:
I haven't seen an `rls.toml` before - would you mind sharing a link to some docs where I can read more?

Reply:
Oh no! My apologies. Shouldn't have let that get committed. Will fix!   Also, this is how I have my atom extension setup https://atom.io/packages/ide-rust , when I am using Atom. Seems most ides have issues with rls and complex projects, variable rust versions. So I switch as required, they all seem to be constantly broken in some way.l